modules/cli/src/main/java/org/apache/ignite/cli/deprecated/spec/ClusterReplCommandSpec.java

```diff
@@ -84,7 +84,7 @@ public static class InitClusterCommandSpec extends BaseCommand {
 
         /** {@inheritDoc} */
         @Override
-        public void run() {
+        public Integer call() {
```

Comment:
Does exit code make any sense in REPL? If no then I don't think we even have to return it.

Reply:
No it doesn't. I'll split the commands so the REPL commands will stay Runnables.